src/components/common/footer.js

```diff
@@ -0,0 +1,220 @@
+import { Link } from 'react-router-dom/cjs/react-router-dom.min';
+import Logo from '../../images/ndc.png';
+
+const navigation = {
+  usefullLinks: [
+    {
+      name: 'I AM HUMAN DOCS',
+      href: 'https://i-am-human.gitbook.io/i-am-human-docs/',
+    },
+    {
+      name: 'Feedback Form',
+      href: 'https://hr6bimbyqly.typeform.com/to/wVhraeUG',
+    },
+    {
+      name: 'Community Application for SBT',
+      to: '/community-application',
+    },
+  ],
+  support: [
+    { name: 'Pricing', href: '#' },
+    { name: 'Documentation', href: '#' },
+    { name: 'Guides', href: '#' },
+    { name: 'API Status', href: '#' },
+  ],
+  company: [
+    { name: 'About', href: '#' },
+    { name: 'Blog', href: '#' },
+    { name: 'Jobs', href: '#' },
+    { name: 'Press', href: '#' },
+    { name: 'Partners', href: '#' },
+  ],
+  legal: [
+    { name: 'Claim', href: '#' },
+    { name: 'Privacy', href: '#' },
+    { name: 'Terms', href: '#' },
+  ],
+  social: [
```

Comment:
where do we show this social?
I am not able to check it on UI

Reply:
This code is not really being used right now , removing